src/state/machines/turn.ts

```diff
@@ -442,10 +440,10 @@ export const TurnMachine = setup({
           player.hand = concat(player.hand, player.deck.slice(0, remainingDraw));
           player.deck = player.deck.slice(remainingDraw);
         }
-        draft.turn.phase = "draw";
+        // draft.turn.phase = "draw";
```

Comment:
Should we remove this permanently since it is not being used anymore?

Reply:
I am not exactly sure how your changes affect this, before this PR it was definitely used. It helps the condition checker to determine where the state should go to next. Line 657 has a guard that checks this phase property, so I am not sure if it is still working or not